test/unit/Signer/RsaTest.php

```diff
@@ -18,10 +18,10 @@
     /**
      * @test
      *
-     * @requires extension openssl
-     *
      * @covers \Lcobucci\JWT\Signer\Rsa::sign
      * @covers \Lcobucci\JWT\Signer\Rsa::validateKey
+     * @covers \Lcobucci\JWT\Signer\Rsa::getKeyType
+     * @covers \Lcobucci\JWT\Signer\OpenSSL
```

Comment:
This is weird, I know... we should simply use OpenSSL and not cover it here. Will open an issue to address this.